embedding/csharp/IPluginRegistry.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+using System;
+
+namespace Tizen.Flutter.Embedding
+{
+    public interface IPluginRegistry
+    {
+        public IntPtr GetRegistrarForPlugin(string pluginName);
```

Comment:
I think it would be better to add the ```abstract``` keyword

```suggestion
        public abstract IntPtr GetRegistrarForPlugin(string pluginName);
```

Reply:
In C#, as far as I know, all methods in interfaces are abstract by default and it's illegal to use the `abstract` keyword.

Reply:
Um... abstract keyword in method declaration, I remember that was a keyword that forced implementation like pure virtual function in c++.
I'm sorry if I was wrong :) and If so, don't mind this opinion

Reply:
>  all methods in interfaces are abstract by default

oh.. I understood